rust/src/table_state.rs

```diff
@@ -283,6 +283,7 @@ impl DeltaTableState {
         require_files: bool,
     ) -> Result<(), ApplyLogError> {
         match action {
+            action::Action::cdc(_v) => {}
```

Comment:
Ought to mention this is temporarily being ignored.

```suggestion
            // TODO: optionally load CDC into TableState
            action::Action::cdc(_v) => {}
```